### PR TITLE
QA-14945: Fix dataStore health probe always green

### PR DESCRIPTION
**Pull request by jahia-carp**
https://jira.jahia.org/browse/QA-14945

### CODE REVIEW

**Review by dgriffon — APPROVED** [review]

File: .github/workflows/manual-run.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ jobs:
         with:
           module_id: server-avaiblaility-manager
           testrail_project: Server Availability Manager
+          timeout_minutes: 20
```
Comment (Fgerthoffert) [inline .github/workflows/manual-run.yml:34]: In manual run you should also add the timeout on at the job level (timeout-minutes: 30)

File: tests/provisioning-manifest-build.yml [hunk 2]
```diff
@@ -1 +1,6 @@
 - karafCommand: "bundle:list"
+# uninstall only if it exists, otherwise it throws an error
+# need to hardcode version for uninstall
+- if: org.jahia.osgi.BundleUtils.getBundleBySymbolicName('server-availability-manager',null) != null
```
Comment (Fgerthoffert) [inline tests/provisioning-manifest-build.yml:4]: What was the issue with requiring an install of a newer version on top of the one embedded inside Jahia?

Why wasn't it sufficient to just install it as part of the regular workflow.

**Review by Fgerthoffert — CHANGES_REQUESTED** [review]

File: tests/provisioning-manifest-build.yml (hunk 2 above)
Comment (jahia-carp) [reply tests/provisioning-manifest-build.yml:4]: The issue with requiring an install of a newer version on top of the one embedded inside Jahia stems from the need to thoroughly test our probes. One of our tests involves ensuring that everything is functioning correctly with a severity level set to "low." In this context, the ModuleStateProbe plays a crucial role as it returns a status of "YELLOW" if at least one module is not started.

File: .github/workflows/manual-run.yml (hunk 1 above)
Comment (jahia-carp) [reply .github/workflows/manual-run.yml:35]: Fixed, thanks

File: tests/provisioning-manifest-build.yml (hunk 2 above)
Comment (jahia-carp) [reply tests/provisioning-manifest-build.yml]: You're right, I hadn't thought of this way of doing things, it's actually better to define this in env.run.sh. 
Fixed, thanks for the feedbacks :-)

**Review by Fgerthoffert — APPROVED** [review]

**Pull request merged**